annileen/engine/cubemap.ixx

```diff
@@ -40,5 +40,7 @@ namespace annileen
 
 	Cubemap::~Cubemap()
 	{
+		// TODO: remove
+		std::cout << "Cubemap destroyed." << std::endl;
```

Comment:
is that supposed to be removed?

Reply:
Yes. I'm adding these comments for debugging reasons for now. I'll go over those TODOs later and delete them.